### PR TITLE
cython-lint for schemes

### PR DESCRIPTION
**Pull request by fchapoton**
fixing a few more cython-lint suggestions in schemes

```cython-lint --ignore=E501,E231,E265 src/sage/schemes/```

### :memo: Checklist

- [x] The title is concise and informative.
- [x] The description explains in detail what this PR is about.

### CODE REVIEW

File: src/sage/schemes/elliptic_curves/mod_sym_num.pyx [hunk 1]
```diff
@@ -1812,7 +1815,7 @@ cdef class ModularSymbolNumerical:
             -2.265525136998248e-05,
             2.3248943281270047e-06]
         """
-        #verbose("       enter _kappa with m=%s, z=%s and eps=%s"%(m,z,eps),
+        #verbose("       enter _kappa with m=%s, z=%s and eps=%s" % (m,z,eps),
```
Comment (dcoudert) [inline src/sage/schemes/elliptic_curves/mod_sym_num.pyx]: space before verbose ? (several places)

File: src/sage/schemes/elliptic_curves/mod_sym_num.pyx [hunk 2]
```diff
@@ -2372,16 +2376,16 @@ cdef class ModularSymbolNumerical:
         This goes via i `\infty`::
 
             sage: M = ModularSymbolNumerical(EllipticCurve("5077a1"))
-            sage: M._transportable_approx( 0/1, -35/144, 0.001) #abs tol 1e-11
+            sage: M._transportable_approx(0/1, -35/144, 0.001) #abs tol 1e-11
```
Comment (dcoudert) [inline src/sage/schemes/elliptic_curves/mod_sym_num.pyx]: `#abs` -> `# abs` (several places), and may be an extra space before `#` ?

File: src/sage/schemes/elliptic_curves/mod_sym_num.pyx [hunk 3]
```diff
@@ -2372,16 +2376,16 @@ cdef class ModularSymbolNumerical:
...
+            sage: M._from_r_to_rr_approx(0/1, -35/144, 0.001) # abs tol 1e-10
             -6.22753204310913 - 1.31710951034592e-8*I
 
         While this one goes via 0::
 
-            sage: M._transportable_approx( 0/1, -7/31798, 0.001) #abs tol 1e-11
+            sage: M._transportable_approx(0/1, -7/31798, 0.001) #abs tol 1e-11
             -7.01577418382726e-9 - 7.40274138232394*I
-            sage: M._from_r_to_rr_approx( 0/1, -7/31798, 0.001) #abs tol 1e-5 #long time
+            sage: M._from_r_to_rr_approx(0/1, -7/31798, 0.001) #abs tol 1e-5 #long time
```
Comment (dcoudert) [inline src/sage/schemes/elliptic_curves/mod_sym_num.pyx]: `#long` -> `# long`

File: src/sage/schemes/elliptic_curves/mod_sym_num.pyx [hunk 4]
```diff
@@ -2933,8 +2937,8 @@ cdef class ModularSymbolNumerical:
         oi = proj_normalise(self._N_E, -u, v, &un, &vn)
         c.set_cache(sign*res, un, vn, sign)
 
-        # (u:v) + ( u+v:-u) +(v,-u-v) = 0
-        # is ( u+v:-u) already computed, we set the third
+        # (u:v) + (u+v:-u) +(v,-u-v) = 0
+        # is (u+v:-u) already computed, we set the third
         oi = proj_normalise(self._N_E, u+v, -u, &un, &vn)
         if c.is_in_cache(un,vn,sign):
```
Comment (dcoudert) [inline src/sage/schemes/elliptic_curves/mod_sym_num.pyx]: spaces between parameters

**Review by dcoudert — CHANGES_REQUESTED** [review]

**Comment by fchapoton** [comment]
ok, voila. Merci pour la relecture, David. Je ne pretends pas avoir tout corrigé, loin de la.

**Review by dcoudert — COMMENTED** [review]
LGTM. However, this PR is not yet in `needs review`. Is it ready ?

**Comment by fchapoton** [comment]
ben, oui, mais l'integration continue est totalement cassée et tout le monde s'en fiche..

**Review by dcoudert — APPROVED** [review]
So then, LGTM.

**Pull request merged**